src/enums/routeNames.ts

```diff
@@ -4,7 +4,10 @@ export enum RouteNames {
   CHANGE = 'change',
   CONVERSION = 'conversion',
   CORRECTION = 'correction',
-  RESTORATION = 'restoration',
+  RESTORATION_FULL = 'fullRestoration',
+  RESTORATION_LIMITED = 'limitedRestoration',
```

Comment:
This app doesn't handle these 2 filings, so you don't need these routes.

Reply:
Good call.  I'll remove them